### DISCUSSION TITLE
Pycaret with uber / manifold

### DISCUSSION THREAD

**Discussion by pit9921** [pycaret/pycaret#1231] [Ideas]
Hi,

is there any way, how to visualize the performance of a pycaret model with uber manifold?
https://github.com/uber/manifold

Peter

**Comment by pycaret** [comment]
@pit9921 Not yet. Feel free to log an issue and we will tag it as an enhancement.

Thanks for suggesting. Looks like a great project and integration opportunity.

**Comment by TomLucidor** [comment]
Still no response? I guess the library is still on but there might be alternatives after all these years
